### DISCUSSION TITLE
adding to existing project

### DISCUSSION THREAD

**Discussion by rot3r-git** [electron-vite/electron-vite-vue#329] [Q&A]
i just follow the instruction and use : npm i electron-vite -D in a existing project but i got error "Cannot find module 'electron/package.json'".
is there any further instruction that i have to follow?

**Comment by caoxiemeihao** [comment]
The npm `electron-vite` package is not a project under this organization.

**Reply by rot3r-git** [reply]
so can you please guide me how to add electron to existing project?

**Reply by caoxiemeihao** [reply]
You should be use `vite-plugin-electron`
